src/components/header/style.scss

```diff
@@ -0,0 +1,176 @@
+@import '../../scss/color.scss';
+
+.header {
+  background-color: $black;
+  height: 80px;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+
+  &__container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__bars {
+    display: none;
+  }
+}
+
+.navbar {
+  &__ul {
+    display: flex;
+  }
+
+  &__li {
+    margin-right: 30px;
+  }
+
+  &__li:nth-child(4) {
+    margin-right: 10px;
+    margin-left: -20px;
+  }
+
+  &__link {
+    color: $white;
+    font-size: 15px;
+    font-weight: 600;
+    transition: color 0.3s ease;
+    font-family: 'Poppins', sans-serif;
+
+    &:hover {
+      color: $green;
+    }
+  }
+
+  & &__li:nth-child(4) &__link {
+    background-color: $white;
+    height: 1px;
+    width: 15px;
+    display: block;
+    transform: rotate(90deg) translateX(8px);
+  }
+}
+
+.navbar__mobile {
+  display: none;
+}
+
+@media only screen and (max-width: 1150px) {
```

Comment:
kích thước màn hình cũng tạo biến luôn nha: xs, sm, md, lg..

Reply:
@media only screen and (max-width: 1150px)   => @media only screen and (max-width: xs  hay sm ) đng ko a

Reply:
done